<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="MainTab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,0,6">
        <item>
         <layout class="QHBoxLayout" name="connectionHorizontalLayout" stretch="1,1,1,1,3">
          <item>
           <widget class="QLabel" name="connectionIndicator">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="connectionComboBox">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectionButton">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Connexion</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deconnectionButton">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Deconnexion</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="temperatureHorizontalLayout" stretch="1,10">
          <item>
           <widget class="QLabel" name="temperatureLabel">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Température :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="temperatureDisplay">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="humidityHorizontalLayout_" stretch="1,10">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="humidityLabel">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Humidité :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="humididyDisplay">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pressureHorizontalLayout" stretch="1,10">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="pressureLabel">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Pression :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pressureDisplay">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="curvesHorizontalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="consoleTab">
       <attribute name="title">
        <string>Console</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="consoleLabel">
          <property name="text">
           <string>Port COM Console</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="consoleClearButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Clear Console</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="ConsoleTextBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
